definition-map.js

```diff
@@ -5,6 +5,20 @@ const TYPES = {
             contentTypes: "Content Types requested in the Catalog."
         }
     },
+    RusticiCourseLaunchResponse: {
+        metadescription: "Contains data to launch SCORM content through Rustici API.",
+        fields: {
+            courseTitle: "The title of the Topic Or Course.",
+            fullscreenEmbed: "Launches the SCORM content in fullscreen. This is the default behavior for Course. For Topic, this is applicable when the topic has the `Embed SCORM on page` option enabled.",
```

Comment:
Think it might be more fair to say "Informs the client whether the SCORM content should be launched in full screen or windowed." since it's not really up to us how the developer decides to use the `url`.